docs/guides/pagination/overview.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Pagination - Overview
+---
+
+# Pagination
+
+Whenever we deal with lists in GraphQL, we usually need to limit the number of items returned. Surely, we don't want to send massive lists of
+items that take a considerable toll on the server! The goal of this guide is to help you get going fast with pagination!
+
+## Pagination at a glance
+
+We have always dealt with pagination in different situations. Let us take a look at some of the common ways pagination
+can be implemented today!
+
+### Offset based pagination
+
+This pagination style is similar to the syntax we use when looking up database records. Here, the client specifies the number of result to be
+obtained at a time, along with an offset- which usually denotes the number of results to be skipped from the beginning. This type of pagination
+is widely used. Implementing offset-based pagination with an SQL database is straight-forward:
+
+- We count all of the results to determine the total number of pages
+- We use the limit and offset values given to query for the items in the requested page.
+
+Offset based pagination also provides us the ability to jump to a specific page in a dataset.
+
+Let us understand offset based pagination better, with an example. Let us assume that we want to request a list of users, 2 at a time, from a server.
+We start out be sending a request to the server, with the desired limit and offset values.
+
+```json
+{
+  "limit": 2,
+  "offset": 0
+}
+```
+
+The response from the server would be:
+
+```json
+{
+  "users": [
+    {
+      "id": 1,
+      "name": "Norman Osborn",
+      "occupation": "Founder, Oscorp Industries",
+      "age": 42
+    },
+    {
+      "id": 2,
+      "name": "Peter Parker",
+      "occupation": "Freelance Photographer, The Daily Bugle",
+      "age": 16
```

Comment:
The age seems a little low :thinking:

Reply:
Right, he was a photographer in college😅😂 I could change that